### DISCUSSION TITLE
是我的插件有问题吗？提示本地条目、远程条目

### DISCUSSION THREAD

**Discussion by abawu-z** [windingwind/zotero-better-notes#804] [Q&A] [answered]
打开会自动反复跳出这个，提示本地条目、远程条目，点击都没有任何变化，也不会跳转，怎么解决呢，谢谢！
<img width="748" alt="问题" src="https://github.com/windingwind/zotero-better-notes/assets/146628735/789f5b3b-c265-4600-9310-3ddc252724fc">

**Comment by windingwind** [comment, marked as answer]
其他插件（例如Chartero）使用笔记进行数据同步，频繁更新笔记导致本地和云端数据冲突，与本插件无关。请联系对应插件作者。此处关闭

**Comment by chenganggang2020** [comment]
是否安装了Linter for Zotero，可能是他的问题

**Comment by fengyuhang0113** [comment]
请问同学你现在解决这个问题了吗

**Comment by pengyaqian** [comment]
请问你这个问题解决了吗 ？我这边是用了两台电脑登录同一个账号，一个是zotero6，一个是zotero7，您是否也是这种情况？

**Comment by fengyuhang0113** [comment]
uu我去淘宝店铺找了：科研工具箱，让店主帮我解决了🤣你也可以试试

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年05月23日 16:10 |
| 收件人 | windingwind/zotero-better-notes ***@***.***> |
| 抄送人 | fengyuhang0113 ***@***.***>,
Comment ***@***.***> |
| 主题 | Re: [windingwind/zotero-better-notes] 是我的插件有问题吗？提示本地条目、远程条目 (Discussion #804) |

请问你这个问题解决了吗 ？我这边是用了两台电脑登录同一个账号，一个是zotero6，一个是zotero7，您是否也是这种情况？

**Reply by bihu3637** [reply]
您好，店家是如何解决的，是删除了哪些冲突插件？谢谢

**Comment by pengyaqian** [comment]
好的好的，谢谢




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "windingwind/zotero-better-notes"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年5月23日(星期四) 下午5:44
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [windingwind/zotero-better-notes] 是我的插件有问题吗？提示本地条目、远程条目 (Discussion #804)





 uu我去淘宝店铺找了：科研工具箱，让店主帮我解决了🤣你也可以试试
 
 ---- 回复的原邮件 ----
 | 发件人 | ***@***.***&gt; |
 | 发送日期 | 2024年05月23日 16:10 |
 | 收件人 | windingwind/zotero-better-notes ***@***.***&gt; |
 | 抄送人 | fengyuhang0113 ***@***.***&gt;,
 Comment ***@***.***&gt; |
 | 主题 | Re: [windingwind/zotero-better-notes] 是我的插件有问题吗？提示本地条目、远程条目 (Discussion #804) |
 
 请问你这个问题解决了吗 ？我这边是用了两台电脑登录同一个账号，一个是zotero6，一个是zotero7，您是否也是这种情况？

**Comment by fengyuhang0113** [comment]
我后面直接全部文献列表都不显示了，然后店家帮我重新安装了一下🥹

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年09月26日 10:37 |
| 收件人 | windingwind/zotero-better-notes ***@***.***> |
| 抄送人 | fengyuhang0113 ***@***.***>,
Comment ***@***.***> |
| 主题 | Re: [windingwind/zotero-better-notes] 是我的插件有问题吗？提示本地条目、远程条目 (Discussion #804) |

您好，店家是如何解决的，是删除了哪些冲突插件？谢谢
